automated_qc/tree_cover_loss_qc.ipynb

```diff
@@ -0,0 +1,565 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Automated Tree Cover Loss QC"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import numpy as np\n",
+    "import rasterio as rio\n",
+    "import fiona\n",
+    "import pandas as pd\n",
+    "from tqdm.notebook import tqdm\n",
+    "from rasterio.windows import from_bounds\n",
+    "from rasterstats import zonal_stats\n",
+    "from matplotlib import pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# data config\n",
+    "\n",
+    "# input directory with three tiles (tree cover density, tree cover loss, and area) and admin boundary shp\n",
+    "DATA_DIR = 'test'\n",
+    "OUT_DIR = 'output'\n",
+    "\n",
+    "# asset filenames\n",
+    "TCD_FN = 'Hansen_GFC-2019-v1.7_treecover2000_20N_100E.tif'\n",
```

Comment:
Can you reference S3 files instead of local files? Also, could you make it possible to run this against any shapefile area instead of specifically Khammouan? We'll want to run this against all the admin areas we decided to QC